### PR TITLE
docs: update stabilization section

### LINKED ISSUE #4661: proposal: publish release candidate before stabilization
I'd suggest the below steps for the package stabilization process:

- We publish 1.0.0-rc.1 (release candidate 1) after 2 maintainers approved it.
- We announce to the community and the core team that we publish 1.0 after 1 month.
  - (For the core team, add calendar entry of the stabilization date)
- Wait 1 month for feedbacks.
- Handle feedbacks, if any.
- If there's no issue after pre-defined amount of time, we publish 1.0
  - If there's unresolved issue remaining, we move the stabilization date forward or cancel it (depending on the situation).

### PR DESCRIPTION
**Pull request by kt3k**
closes #4661

### CODE REVIEW

**Review by iuioiua — APPROVED** [review]
I rewrote this, using your points, with the aim of improving the flow and clarity of the instructions. Please let me know if you have any issues. But now, to me, LGTM.

File: README.md [hunk 1]
```diff
@@ -64,17 +64,23 @@ documentation:
...
+1. Publish version 1.0.0-rc.1 once meeting the following requirements:
+   1. Approved by at least 2 maintainers. There must be consensus that the API
+      design is satisfactory and unlikely to change in the future.
+   1. 100% documented, passing `deno doc --lint` checks and adhering to the
+      [documentation guidelines](https://github.com/denoland/deno_std/blob/main/.github/CONTRIBUTING.md#documentation).
+   1. Maximum possible test coverage
+   1. No open issues or pull requests that might lead to breaking changes. For
+      example, issues that suggest new non-breaking features are fine to exist
+      at stabilization.
+1. Allow 1 month for 2 other maintainers to review the package and handle any
```
[the remark this answers is not in the document]
Comment (kt3k) [reply README.md:80]: I think no feedback 1 month automatically means the approval.

I also don't want to assign core team people randomly to review tasks.

**Comment by kt3k** [comment]
Landing for now. Let's keep iterating on details.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (iuioiua) [reply README.md]: No feedback after 1 month could also mean no one has looked at the package. I think some degree of approval from the core team should be a requirement, even if it's one other person.

**Comment by iuioiua** [comment]
I think the current wording allows for reviews from the internal team. LGTM.

**Pull request merged**